galaxy.yml.j2

```diff
@@ -13,15 +13,18 @@ repository: {{ collection_repo }}
 issues: {{ collection_repo }}/issues
 build_ignore:
     - galaxy.yml.j2
-    - release.yml
     - .github
+    - .markdownlint.yml
+    - .gitignore
+    - .gitattributes
 license:
     - GPL-3.0-or-later
 tags:
-    - tower
```

Comment:
I would advocate for leaving the `tower` tags in here as this collection can (I think) still be used to deploy Tower and certainly the older versions can be.

Reply:
That makes sense to me. I'll add Tower back in.